ipst.yml

```diff
@@ -193,15 +226,13 @@
     hades_situation_prefix: "{{ user_hades_situation_prefix |default('/DATA/IIDM/FO/2016/01/01') }}"
 
 ## local test situation path
-    hades_situations_src: "{{ user_hades_situations_prefix_src |default('~/tmp/situations/grovslb1/local') + hades_situation_prefix + '/'+ hades_situation_test }}"
-
-
+    hades_situations_src: "{{ user_hades_situations_prefix_src |default('$HOME/tmp/situations/grovslb1/local') + hades_situation_prefix + '/'+ hades_situation_test }}"
```

Comment:
when user_hades_process==True and the hades2LF.tar.gz is in ~/tmp/hades,
Ansible correctly installs the hades binaries & the integration jars. 
However, it fails in the following test because the 'situation' test files are not available.
(As far as I know, 20160101_0030_FO5_FR0.xiidm.gz is not part of any distribution)
This test should be made optional, I think.

Reply:
done